### ISSUE TITLE
Unhandled exception at 0x5526CDC4 (GLedApi.dll) in RGBFusionTool.exe: Stack cookie instrumentation code detected a stack-based buffer overrun.

### ISSUE THREAD

**Issue by windows2000bug** [tylerszabo/RGB-Fusion-Tool#18]
Unhandled exception at 0x5526CDC4 (GLedApi.dll) in RGBFusionTool.exe: Stack cookie instrumentation code detected a stack-based buffer overrun.

Original Location: C:\Program Files (x86)\rgb-fusion\GLedApi.dll

I don't have many other items other than that. This is when I was testing my PS script. IF you need further details, please let me know.

**Comment by tylerszabo** [comment]
Yes, I'd like additional details - though this might be an issue in Gigabyte's code

**Comment by windows2000bug** [comment]
It opened up Studio 2017 and I saved a SLN, do you want me to upload that? It wasn't a the typical "output error to command line" A window popped up that said it died/crashed. Is there anything in particular you need?

**Comment by tylerszabo** [comment]
A dump file would help. I don't think an SLN file would have information that would help debug it.
